client/components/plans/plan-price/index.jsx

```diff
@@ -62,13 +62,30 @@ module.exports = React.createClass( {
 			periodLabel = hasDiscount ? this.translate( 'due today when you upgrade' ) : plan.bill_period_label
 		}
 
-		return (
-			<div className={ hasDiscount ? "plan-price plan-price__discount" : "plan-price" }>
-				<span>{ this.getPrice() }</span>
-				<small className="plan-price__billing-period">
-					{ periodLabel }
-				</small>
-			</div>
-		);
+		if ( site && site.jetpack ) {
+			return (
+				<div className="plan-price">
+					<span className="jetpack-price">{ plan.formatted_original_price }</span>
+					<small className="plan-price__billing-period">
+						{ this.translate( 'cost of individual plugins' ) }
+		      </small>
+		      <span className="jetpack-price">{ this.getPrice() }</span>
+		      <small className="plan-price__billing-period">
+		        { hasDiscount ? this.translate( 'for first year' ) : plan.bill_period_label } (
```

Comment:
Fixed in fa052ee164ece70ae04893b18f216474c54aed2c

I've included the `%` in the string as you suggested. The savings don't have a direct relation to `for first year` (especially since that particular string is only showing if a further discount is being applied). 

However I added a translator's context note to make it clear.